infrastructure-playbooks/purge-cluster.yml

```diff
@@ -52,9 +52,27 @@
   become: true
 
   tasks:
-  - name: detect init system
-    command: ceph-detect-init
-    register: init_system
+    - debug: msg="gather facts on all Ceph hosts for following reference"
+
+    - name: set cluster value if not set
+      set_fact:
+        cluster: ceph
+      when: cluster is not defined
+
+    - name: check if sysvinit
+      stat:
+        path: /etc/rc?.d/S??ceph
```

Comment:
This actually does not work. We need to use the `stat` command instead I'm afraid...